starcheck/src/lib/Ska/Starcheck/Obsid.pm

```diff
@@ -23,14 +23,47 @@ use warnings;
 
 use Inline Python => q{
 import numpy as np
+from astropy.table import Table
+
+from mica.archive import aca_dark
 from chandra_aca.star_probs import guide_count
+from chandra_aca.transform import yagzag_to_pixels, count_rate_to_mag
 import Quaternion
 from Ska.quatutil import radec2yagzag
 import agasc
 
+from proseco.core import ACABox
+from proseco.guide import get_imposter_mags
+
 def _guide_count(mags, t_ccd):
     return float(guide_count(np.array(mags), t_ccd))
 
+def check_hot_pix(idxs, yags, zags, mags, types, t_ccd, date, dither_y, dither_z, dmag=1.0):
+
+    dark = aca_dark.get_dark_cal_image(date=date.decode('ascii'),
```

Comment:
I should probably see about getting the date of the dark cal put out in the processing summary.

Reply:
It is also getting fetched once per obsid in this configuration, but would be a few more steps to set it and pass it per-schedule.

Reply:
With regard to getting the date in the processing summary, @taldcroft , do you want me to head in that direction or leave this alone?   It seems like the implementation would be to fetch the dark cal in starcheck.pl and try to either set as a Python global or pass to Obsid object or check_starcheck_catalog.

Could also do something like use one date for all these calls and then *infer* the dark cal use in starcheck.pl.

Reply:
I think this can wait for a subsequent release, so I recommend to just open an issue.  We might help ourselves out by taking milestoning more seriously, so make a new milestone and tag thusly.  This can help separate things we actually want for a release from, well, stuff that comes up.

Reply:
Sounds good.  Another advantage of calling get_dark_cal_image right here is that we're getting the temperature scaling per obsid without needing to do anything more manually.